app/navigation/local-notification-wrapper.tsx

```diff
@@ -0,0 +1,40 @@
+import { useApolloClient } from "@apollo/client"
+import { useIsAuthed } from "@app/graphql/is-authed-context"
+import { addDeviceToken, hasNotificationPermission } from "@app/utils/notifications"
+import messaging, { FirebaseMessagingTypes } from "@react-native-firebase/messaging"
+import { useEffect } from "react"
+
+export const NotificationWrapper = (): JSX.Element => {
```

Comment:
This doesn't appear to be a wrapper, but just a component that handles the notification logic.

Reply:
any suggestion of name we should give to such component?